server/pr2_moderation.php

```diff
@@ -209,14 +209,14 @@ function demote_moderator($socket, $name) {
 
 	if($from_player->group == 3){
 		$to_player = name_to_player($name);
+		if(isset($to_player) && $to_player->group < 2) {
+			$from_player->write('message`Error: '.$name.' is not a moderator.');
+		}
 		if(isset($to_player) && $to_player->group == 2) {

```

Comment:
Should this be "else if"?

Reply:
It could be. I can change that

Reply:
Oh, I see. Both statements can't be true at the same time. I'm just trying to figure out how this stops two messages from getting sent?

Reply:
If it's left as it is, this is the execution:

 - Demod
 - Check for a player's group and echo an error message if they're not a mod

Which produces two messages:
 - ____ has been demoted.
 - Error: ___ is not a moderator.

So, by changing the order we will prevent the double check and just go with one message.

Reply:
Gotcha, thanks!